tests/integration/conftest.py

```diff
@@ -370,3 +421,73 @@ def is_mysql_ready():
         ),
     )
     await ops_test.model.wait_for_idle()
+
+
+@pytest.fixture(scope="module", name="test_image")
+def image_fixture():
+    """A PNG image that can be used in tests."""
+    image_response = requests.get(
+        "https://s.w.org/style/images/about/WordPress-logotype-wmark.png", timeout=10
+    )
+    assert image_response.status_code == 200
+    return image_response.content
+
+
+@pytest.fixture(scope="module", name="swift_conn")
+def swift_conn_fixture(openstack_environment):
+    """Create a swift connection client."""
+    if openstack_environment is None:
+        return None
+    swift_conn = swiftclient.Connection(
+        authurl=openstack_environment["OS_AUTH_URL"],
+        auth_version="3",
+        user=openstack_environment["OS_USERNAME"],
+        key=openstack_environment["OS_PASSWORD"],
+        os_options={
+            "user_domain_name": openstack_environment["OS_USER_DOMAIN_ID"],
+            "project_domain_name": openstack_environment["OS_PROJECT_DOMAIN_ID"],
+            "project_name": openstack_environment["OS_PROJECT_NAME"],
+        },
+    )
+    return swift_conn
+
+
+@pytest.fixture(scope="module", name="swift_config")
+def swift_config_fixture(request: pytest.FixtureRequest, swift_conn, openstack_environment):
+    """Create a swift config dict that can be used for wp_plugin_openstack-objectstorage_config."""
+    if openstack_environment is None:
+        return None
+    swift_service = swiftclient.service.SwiftService(
+        options=dict(
+            auth_version="3",
+            os_auth_url=openstack_environment["OS_AUTH_URL"],
+            os_username=openstack_environment["OS_USERNAME"],
+            os_password=openstack_environment["OS_PASSWORD"],
+            os_project_name=openstack_environment["OS_PROJECT_NAME"],
+            os_project_domain_name=openstack_environment["OS_PROJECT_DOMAIN_ID"],
+        )
+    )
+    container = f"wordpress_{request.module.__name__}"
+    print(f"Use container {container}")
```

Comment:
Is this intended to be there still?

Reply:
No, I will change this into `logger.info`, thanks!